src/lib/queries/getAccountsTotals.ts

```diff
@@ -15,7 +16,7 @@ export default async function getAccountsTotals(
   const rows: { total: number, accountId: string, mnemonic: string }[] = await Split
     .query(`
       SELECT
-        SUM(cast(splits.quantity_num as REAL) / splits.quantity_denom) as total,
+        ABS(SUM(cast(splits.quantity_num as REAL) / splits.quantity_denom)) as total,
```

Comment:
we don't gain anything from INCOME and LIABILITY accounts being negative so converting to ABS here rather than having specific logic in the components to display one way or the other.